src/features/metadata-instance-editor/Instance.js

```diff
@@ -531,7 +521,8 @@ class Instance extends React.PureComponent<Props, State> {
      */
     canEdit(): boolean {
         const { canEdit, onModification, onRemove, onSave }: Props = this.props;
-        return (
+
+        return !!(
```

Comment:
I don't think we need this since each of the conditions between the && will return booleans

Reply:
Removed the !!